bin/aws-sso-extensions-for-enterprise.ts

```diff
@@ -125,11 +166,30 @@ function getConfig() {
         unparsedEnv["PipelineSettings"],
         "OrgMainAccountId"
       ),
-      RepoArn: ensureString(unparsedEnv["PipelineSettings"], "RepoArn"),
-      RepoBranchName: ensureString(
+      RepoType: ensureValidString(unparsedEnv["PipelineSettings"], "RepoType", [
+        "CODECOMMIT",
```

Comment:
This is conflicting with the value in the `env.yaml` file (`CodeCommit`) - causing the security-checks to fail. Could you please standardize on a case and add the allowed values as an inline comment in the `env.yaml`?

Reply:
I am not sure why this is erroring out on the CI spec. The utility functions would do a type conversion , precisely to take care of scenarios where config would have lower/uper/camel case. Can you let me know what the failure here is?

Reply:
Sorry, user error - I added `RepoName` to the wrong section.